resources/icons/16px/sort-asc-filter.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M16 2C16 2.28 15.89 2.53 15.71 2.7L12 6.41V10C12 10.28 11.89 10.53 11.71 10.71L9.71 12.71C9.53 12.89 9.28 13 9 13C8.45 13 8 12.55 8 12V6.41L4.29 2.71C4.11 2.53 4 2.28 4 2C4 1.45 4.45 1 5 1H15C15.55 1 16 1.45 16 2Z" fill="#5F6B7C"/>
+<path fill-rule="evenodd" clip-rule="evenodd" d="M5 12.0051C4.72 12.0051 4.47 11.8953 4.29 11.7155L4 11.4258V15.0014C4 15.5507 3.55 16.0001 3 16.0001C2.45 16.0001 2 15.5507 2 15.0014V11.4159L1.71 11.7055C1.53 11.8853 1.28 11.9951 1 11.9951C0.45 11.9951 0 11.5457 0 10.9964C0 10.7167 0.11 10.467 0.29 10.2873L2.29 8.28976C2.47 8.10999 2.72 8.00012 3 8.00012C3.28 8.00012 3.53 8.10999 3.71 8.28976L5.71 10.2873C5.89 10.467 6 10.7167 6 10.9964C6 11.5557 5.55 12.0051 5 12.0051Z" fill="#5F6B7C"/>
+</svg>
```

Comment:
Can we remove the `width`, `height`, `fill`, `fill-rule`, and `clip-rule` attributes from these SVGs? They aren't necessary for the purposes of including in the icon library.

```svg
<svg viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg">
<path d="M16 2C16 2.28 15.89 2.53 15.71 2.7L12 6.41V10C12 10.28 11.89 10.53 11.71 10.71L9.71 12.71C9.53 12.89 9.28 13 9 13C8.45 13 8 12.55 8 12V6.41L4.29 2.71C4.11 2.53 4 2.28 4 2C4 1.45 4.45 1 5 1H15C15.55 1 16 1.45 16 2Z"/>
<path d="M5 12.0051C4.72 12.0051 4.47 11.8953 4.29 11.7155L4 11.4258V15.0014C4 15.5507 3.55 16.0001 3 16.0001C2.45 16.0001 2 15.5507 2 15.0014V11.4159L1.71 11.7055C1.53 11.8853 1.28 11.9951 1 11.9951C0.45 11.9951 0 11.5457 0 10.9964C0 10.7167 0.11 10.467 0.29 10.2873L2.29 8.28976C2.47 8.10999 2.72 8.00012 3 8.00012C3.28 8.00012 3.53 8.10999 3.71 8.28976L5.71 10.2873C5.89 10.467 6 10.7167 6 10.9964C6 11.5557 5.55 12.0051 5 12.0051Z"/>
</svg>
```